cmd/lncli/commands.go

```diff
@@ -2476,9 +2476,18 @@ func sendToRoute(ctx *cli.Context) error {
 			"from incoming array of routes: %v", err)
 	}
 
+	if len(routes.Routes) == 0 {
```

Comment:
Consider just making this a single route argument

Reply:
In other PRs @roasbeef brought up the argument of keeping the cli arguments unchanged to not break scripts. I don't have a strong opinion on the `lncli` interface myself, so kept it compatible here.